src/components/Modals/Onboarding/OnboardingReview.vue

```diff
@@ -40,13 +40,17 @@
           <div class="onboarding-selectWrapper-review">
             <label class="onboarding-label">College*</label>
             <div>
-              <label class="onboarding-label--review">{{ collegeText }}</label>
+              <label id="onboarding-college" class="onboarding-label--review">{{
```

Comment:
let's avoid id so that they won't accidentally collide to introduce a11y problems. Usually the convention is to use `data-test-id`: https://docs.cypress.io/api/commands/get.html#Find-5-elements-with-the-given-data-attribute